lib/core/src/Cardano/Wallet/Network/Ports.hs

```diff
@@ -26,16 +26,20 @@ module Cardano.Wallet.Network.Ports
     -- * Helpers
     , waitForPort
     , unsafePortNumber
+    , findPort
+    , randomUnusedTCPPorts
```

Comment:
Ideally we wouldn't have test util functions in the core library.
But I think you couldn't put it into the core-integration library due to a package cycle. Is that correct?

Reply:
Actually no. I needed to share them and `Cardano.Wallet.Network.Ports` felt like a logical place to put them. I see your point about the "non production" aspect of these two. I believe they could fit in core-integration as well :thinking: